pkg/autoscaler/statforwarder/doc.go

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*
+
+Package statforwarder provides a forwarder which can be used to forward autoscaler
+statistics to the right Autoscaler pod if Autoscaler is set up with multiple
+pods and running in Leader Election mode.
+
+*/
```

Comment:
for consistency with other `doc.go`s

```suggestion
// Package statforwarder provides a forwarder which can be used to forward autoscaler
// statistics to the right Autoscaler pod if Autoscaler is set up with multiple
// pods and running in Leader Election mode.
```